src/applications/mhv/medical-records/tests/e2e/medical-records-understanding-your-results-chem-hem-detail-page.cypress.spec.js

```diff
@@ -0,0 +1,43 @@
+import MedicalRecordsSite from './mr_site/MedicalRecordsSite';
+import LabsAndTestsListPage from './pages/LabsAndTestsListPage';
+import ChemHemDetailsPage from './pages/ChemHemDetailsPage';
+
+describe('Medical Records Understanding Your Results Detail Page', () => {
+  const site = new MedicalRecordsSite();
+
+  before(() => {
+    site.login();
+    cy.visit('my-health/medical-records/labs-and-tests');
+  });
+
+  it('Understanding Your Results Chem/Hem Detail Page', () => {
+    // Given As a Medical Records User I wanted to Navigate to "Chemistry And Hematology" Detail Page
+    LabsAndTestsListPage.clickLabsAndTestsDetailsLink(0);
+    ChemHemDetailsPage.verifyLabName(
+      'POTASSIUM:SCNC:PT:SER/PLAS:QN:, SODIUM:SCNC:PT:SER/PLAS:QN:',
+    );
+    // When I want to get "help to be able to understand results" of my MR data
+    ChemHemDetailsPage.verifyEpnadUnderstandResultsBtton();
+    ChemHemDetailsPage.clickExpnadUnderstandResultsBtton();
```

Comment:
don't retest lab name here. We should always minimize duplication of testing as a rule.